cdbot/cogs/cyber.py

```diff
@@ -133,7 +136,7 @@ def __init__(self, bot: Bot):
             level_num = int(base)
             base = "Headquarters"
         if level_num == 13 and challenge_num == 1:
-            content = "13.1 is a No Flag Zone™ 🙅⛔⚔️"
+            content = "13.1 is a No Flag Zone™ ðﾟﾙﾅ⛔⚔️"
```

Comment:
As a note; the formatting of an emoji here appears to have been changed / become invalid.